### PR TITLE
user-changes.md: document SPDX license expressions

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**

### CODE REVIEW

File: doc/user-changes.md [hunk 1]
```diff
@@ -57,6 +57,23 @@ an index containing unknown values.
...
+expression](https://spdx.org/licenses/). Custom license identifiers are
+accepted with the format: `custom-[0-9a-zA-Z.-]+`.
+
+Example: 
+```toml 
+licenses = "MIT OR custom-my-own-license"
+```
+
+For the `1.x` release, usage of the previous `licenses` format is obsolete and
+will trigger a warning. In furtures major releases this format will not be
```
Comment (mosteo) [inline doc/user-changes.md]: Typo in "future"

**Comment by mosteo** [comment]
I guess the base branch should be `release/1.0`? I haven't yet split the post-1.0 version so this way it will go in in both branches.

**Comment by Fabien-Chouteau** [comment]
It should go in both branches `master` and `1.0`.

**Comment by Fabien-Chouteau** [comment]
I rebased on `release/1.0` and we can then merge the two last PR in `master`.

**Review by mosteo — APPROVED** [review]

**Pull request merged**